clippy_lints/src/tabs_in_doc_comments.rs

```diff
@@ -89,48 +89,69 @@ impl EarlyLintPass for TabsInDocComments {
     }
 }
 
-fn get_chunks_of_tabs(the_str: &str) -> Vec<(usize, usize)> {
-    let mut spans: Vec<(usize, usize)> = vec![];
-    let mut current_start = 0;
+///
+/// scans the string for groups of tabs and returns the start(inclusive) and end positions
+/// (exclusive) of all groups
+/// e.g. "sd\tasd\t\taa" will be converted to [(2, 3), (6, 8)] as
+///       012 3456 7 89
+///         ^-^  ^---^
+fn get_chunks_of_tabs(the_str: &str) -> Vec<(u32, u32)> {
+    let line_length_way_to_long = "line length too long for this diagnosis";
```

Comment:
`"doc comment longer than 2^32 chars"` would be a better (more specific) `expect` message, IMO

Reply:
done :)